### ISSUE TITLE
clicking on sidebar navigation numerals generates js error

### ISSUE THREAD

**Issue by xolotl** [hypothesis/client#562]
### Steps to reproduce
1. Go to a page with annotations.
2. Open JS console.
3. Open the sidebar.
4. Click on the numeral 1 at the left of the sidebar that represents the current annotation in view.
5. Note error is sidebar:
```
bucket-bar.coffee:1 Uncaught ReferenceError: annotator is not defined
    at HTMLDivElement.<anonymous> (bucket-bar.coffee:1)
    at HTMLDivElement.dispatch (jquery.slim.js:5206)
    at HTMLDivElement.v.handle (jquery.slim.js:5014)
(anonymous) @ bucket-bar.coffee:1
dispatch @ jquery.slim.js:5206
v.handle @ jquery.slim.js:5014
```

### Expected behaviour
No error should be generated. Even better, numerals for annotations in view that do not perform and action when clicked should not even be clickable.

### Actual behaviour
See steps to reproduce above.

### Browser/system information
Chrome Version 61.0.3163.100 (Official Build) (64-bit)
Mac OS 10.12.6

### Additional details
Based on recreating a user report:
https://hypothesis.zendesk.com/agent/tickets/1684

![henry_a__giroux___thinking_dangerously__the_role_of_higher_education_in_authoritarian_times](https://user-images.githubusercontent.com/997548/32014429-edcd7758-b972-11e7-998f-542b3119f336.png)

**Comment by felliott** [comment]
We're seeing this in local development with the OSF, too.  It seems to happen when there is only one bucket.  I'm not familiar with coffeescript, but should [this line](https://github.com/hypothesis/client/blob/master/src/annotator/plugin/bucket-bar.coffee#L254) be `@annotator.selectAnnotations` instead of `annotator.selectAnnotations`?

**Comment by lkraav** [comment]
Is it safe to close https://github.com/hypothesis/browser-extension/issues/69 in favor of this?

**Comment by robertknight** [comment]
Fixed in 3c7e9ddd0b44a18519a9bc7a214e4fe41d4d89e4.
